theano/gpuarray/nerv.py

```diff
@@ -1,193 +1,7 @@
-        for name in self.KERN_NAMES:
-            codel.append(Gemm16.cubin_to_code(name))
-        return '\n'.join(codel)
-
-    def c_support_code_struct(self, node, nodename):
-        codel = []
-        for name in self.KERN_NAMES:
-            codel.append("GpuKernel k_{0};".format(name))
-        codel.append(super(Gemm16, self).c_support_code_struct(node, nodename))
-        return '\n'.join(codel)
-
-    def c_init_code_struct(self, node, nodename, sub):
-        codel = [super(Gemm16, self).c_init_code_struct(node, nodename, sub)]
-        for name in self.KERN_NAMES:
-            codel.append("memset(&k_{0}, 0, sizeof(GpuKernel));".format(name))
-        codel.append("const char *bcode;")
-        codel.append("size_t sz;")
-        codel.append("PyGpuContextObject *c = %s;" % (sub['params'],))
-        codel.append("int types[13] = {GA_BUFFER, GA_BUFFER, GA_BUFFER, "
-                     "GA_BUFFER, GA_INT, GA_INT, GA_INT, GA_INT, GA_INT, "
-                     "GA_INT, GA_FLOAT, GA_FLOAT, GA_INT};")
-        for name in self.KERN_NAMES:
-            codel.append(self.init_gpukernel(name, sub['fail']))
-        return '\n'.join(codel)
-
-    def c_cleanup_code_struct(self, node, nodename):
-        codel = []
-        for name in self.KERN_NAMES:
-            codel.append("GpuKernel_clear(&k_{0});".format(name))
-        return '\n'.join(codel)
-
-
-@opt.register_opt('fast_compile')
-@opt.op_lifter([tensor.Dot])
-@opt.register_opt2([tensor.Dot], 'fast_compile')
-def local_gpua_dot_to_gemm16(op, ctx_name, inputs, outputs):
-    if nerv is None:
-        return
-    A = inputs[0]
-    B = inputs[1]
-    if (A.ndim == 2 and B.ndim == 2 and
-            A.dtype == 'float16' and B.dtype == 'float16'):
-        fgraph = getattr(outputs[0], 'fgraph', None)
-        C = GpuAllocEmpty('float16', ctx_name)(
-            shape_i(A, 0, fgraph), shape_i(B, 1, fgraph))
-        return Gemm16()(C, 1.0, A, B, 0.0)
-
-
-@opt.register_opt()
-@alpha_merge(Gemm16, alpha_in=1, beta_in=4)
-def local_gemm16_alpha_merge(node, *inputs):
-    return [Gemm16(relu=node.op.relu)(*inputs)]
-
-
-@opt.register_opt()
-@output_merge(Gemm16, alpha_in=1, beta_in=4, out_in=0)
-def local_gemm16_output_merge(node, *inputs):
-    return [Gemm16(relu=node.op.relu)(*inputs)]
-
-
-@local_optimizer([Gemm16], inplace=True)
-def local_gemm16_inplace(node):
-    if type(node.op) != Gemm16 or node.op.inplace:
-        return
-    inputs = list(node.inputs)
-    C = inputs[0]
-    if (C.owner and
-            isinstance(C.owner.op, GpuAllocEmpty) and
-            len(C.clients) > 1):
-        inputs[0] = C.owner.op(*C.owner.inputs)
-    return [Gemm16(relu=node.op.relu, inplace=True)(*inputs)]
-
-optdb.register('local_gemm16_inplace',
-               tensor.opt.in2out(local_gemm16_inplace,
-                                 name='local_gemm16_inplace'),
-               70.0, 'fast_run', 'inplace', 'gpuarray')
+raise SkipTest("You are importing theano.gpuarray.nerv. "
```

Comment:
I apparently didn't catch that sandbox.cuda uses SkipTest, but yeah, ImportError is better.

If that makes some tests fail, then we can fix them.